category.php

```diff
@@ -0,0 +1,116 @@
+<?php
+get_header();
+?>
+
+<!-- main content -->
+
+<main id="main-content">
+
+<?php
+  $tv = get_category_by_slug('tv');
+  $category = get_category(get_query_var('cat'));
+?>
+
+  <!-- main posts loop -->
+  <section id="posts" class="container">
+
+    <div class="row">
+      <div class="col col24 margin-bottom-basic">
+        <h4><a href="<?php echo get_category_link($category->term_id); ?>"><?php echo $category->name; ?></a></h4>
+      </div>
+    </div>
+
+<?php
+  if ($tv->term_id === $category->term_id || $tv->term_id === $category->category_parent) {
```

Comment:
but why tho? the layout is the same otherwise so all that means is that if the category page needs to be edited then 2 templates have to be edited